ng/js/controllers/new_posting.js

```diff
@@ -0,0 +1,270 @@
+require('angular').module('ays')
+  .controller('CreatePostingController', function ($scope, $sails, $timeout, $state) {
+    $scope.categoryOptions = [];
+    $scope.categoryDescription = '';
+    $scope.titleErrorMessages = [];
+    $scope.priceErrorMessages = [];
+    $scope.descriptionErrorMessages = [];
+    $scope.categoryErrorMessages = [];
+
+    $scope.allInputValidated = false;
+    $scope.titleValidated = false;
+    $scope.pricesValidated = false;
+    $scope.descriptionValidated = false;
+    $scope.categoryValidated = false;
+    $scope.isSubmitting = false;
+
+    // Populate category options based on what server has
+    $sails.get('/api/category')
+      .then(function (response) {
+        var i = 0;
+        for (i = 0; i < response.data.length; i++) {
+          $scope.categoryOptions.push(response.data[i].description);
+        }
+      });
+
+    $scope.newPosting = {
```

Comment:
Yeah, the name was something I found online and I forgot to change it to something that makes sense.